qdrant-landing/content/articles/qdrant-1.2.x.md

```diff
@@ -0,0 +1,202 @@
+were highly requested by our users. If you want to shape the development of the Qdrant vector
+database, please [join our Discord community](https://qdrant.to/discord) and let us know
+how you use it!
+
+## New features
+
+As usual, a minor version update of Qdrant brings some interesting new features. We love your 
+feedback and tried to include the most requested features, as reported by our community.
+
+### Product Quantization
+
+The primary focus of Qdrant was always performance. That's why we built it in Rust, but we were 
+always concerned about making vector search affordable. From the very beginning, Qdrant offered 
+support for disk-stored collections, as storage space is way cheaper than memory. That's also 
+why we have introduced the [Scalar Quantization](/articles/scalar-quantization) mechanism recently, 
+which makes it possible to reduce the memory requirements by up to four times.
+
+Today, we are bringing a new quantization mechanism to life. A separate article on 
+[Product Quantization](/articles/product-quantization) describes that feature in more detail. 
+In a nutshell, you can **reduce the memory requirements by up to 64 times**!
+
+### Optional named vectors
+
+Qdrant has been supporting multiple named vectors per point for quite a long time. Those may have 
+utterly different dimensionality and distance functions used to calculate similarity. Having multiple 
+embeddings per item is an essential real-world scenario. For example, you might be encoding textual 
+and visual data using different models. Or you might be experimenting with different models but 
+don't want to make your payloads redundant by keeping them in separate collections.
+
+![Optional vectors](/articles_data/qdrant-1.2.x/optional-vectors.png)
+
+However, up to the previous version, we requested you provide all the vectors for each point. There 
+have been many requests to allow nullable vectors, as sometimes you cannot generate an embedding or 
+simply don't want to for reasons we don't need to know.
+
+### Grouping requests
+
+Embeddings are great for capturing the semantics of the documents, but we rarely encode larger pieces 
+of data into a single vector. Having a summary of a book may sound attractive, but in reality, we 
+divide it into paragraphs or some different parts to have higher granularity. That pays off when we 
+perform the semantic search, as we can return the relevant pieces only. That's also how modern tools 
+like Langchain process the data. The typical way is to encode some smaller parts of the document and 
+keep the document id as a payload attribute.
+
+![Query without grouping request](/articles_data/qdrant-1.2.x/without-grouping-request.png)
+
+There are, however, some cases, in which we want to find the relevant parts, but no more than just 
+a single result per document. Up till now, we had to implement such a mechanism on the client side 
```

Comment:
Not sure if we need to mention that we can generate group of different sizes, not only the top 1

Reply:
Good point, I'll mention that!